core/src/main/java/org/web3j/protocol/http/HttpService.java

```diff
@@ -61,6 +65,11 @@
     private static final ConnectionSpec INFURA_CIPHER_SUITE_SPEC = new ConnectionSpec
             .Builder(ConnectionSpec.MODERN_TLS).cipherSuites(INFURA_CIPHER_SUITES).build();
 
+    /**
+     * The list of {@link ConnectionSpec} instances used by the connection.
+     */
+    private static final List<ConnectionSpec> CONNECTION_SPEC_LIST = Arrays.asList(INFURA_CIPHER_SUITE_SPEC, CLEARTEXT);
```

Comment:
Line should be less than 100 chars

Reply:
Good spot, I thought I bumped it up to 120 chars in the 3.6 release. I'll change this line to make it 100.